.github/workflows/gpuci.yml

```diff
@@ -57,33 +66,43 @@ jobs:
       # `bash -c "sleep infinity"` keeps the container running indefinitely without exiting
     - name: Run Docker container
       run: |
-        docker run --gpus all --name nemo-curator-container -d nemoci.azurecr.io/nemo_curator_container:${{ github.run_id }} bash -c "sleep infinity"
+        docker run --gpus all --name $CONTAINER_NAME -d nemoci.azurecr.io/nemo_curator_container:${{ github.run_id }} bash -c "sleep infinity"
 
       # Expect `whoami` to be "azureuser"
       # Expect `nvidia-smi` to show our 2 A100 GPUs
     - name: Check GPUs
       run: |
         whoami
-        docker exec nemo-curator-container nvidia-smi
+        docker exec $CONTAINER_NAME nvidia-smi
 
       # In the virtual environment (called "curator") we created in the container,
       # list all of our packages. Useful for debugging
     - name: Verify installations
       run: |
-        docker exec nemo-curator-container pip list
+        docker exec $CONTAINER_NAME pip list
 
       # In the virtual environment (called "curator") we created in the container,
       # run our PyTests marked with `@pytest.mark.gpu`
       # We specify the `rootdir` to help locate the "pyproject.toml" file (which is in the root directory of the repository),
       # and then the directory where the PyTests are located
     - name: Run PyTests with GPU mark
       run: |
-        docker exec nemo-curator-container pytest -m gpu --rootdir /opt/NeMo-Curator /opt/NeMo-Curator/tests
+        docker exec $CONTAINER_NAME pytest -m gpu --rootdir /opt/NeMo-Curator /opt/NeMo-Curator/tests
 
       # After running `docker stop`, the container remains in an exited state
       # It is still present on our system and could be restarted with `docker start`
       # Thus, we use `docker rm` to permanently removed it from the system
     - name: Cleanup
       if: always()
       run: |
-        docker stop nemo-curator-container && docker rm nemo-curator-container
+        docker stop $CONTAINER_NAME && docker rm $CONTAINER_NAME
+
+      # TODO Add notification step for nightly failures
+      # - name: Notify on failure
+      #   if: failure() && github.event_name == 'schedule'
+      #   uses: slackapi/slack-github-action@v1
+      #   with:
+      #     channel-id: 'xyz'
+      #     slack-message: "Nightly GPU tests failed! Check: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+      #   env:
+      #     SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }} # add secret somewhere
```

Comment:
Just wanted to confirm if there is anywhere else the logs are viewable (besides Slack for failures)? Maybe even a way for me to subscribe to look at it every time it is run?

Reply:
I believe we should be able to see the logs here https://github.com/NVIDIA/NeMo-Curator/actions/workflows/gpuci.yml

I'm not sure how we can subscribe tbh.

Reply:
I see, thanks! I was also wondering is there something blocking the Slack notifications atm?